tsconfig.json

```diff
@@ -7,8 +7,18 @@
       "skipLibCheck": true,
       "forceConsistentCasingInFileNames": true,
       "allowImportingTsExtensions": true,
-      "noEmit": true
+      "noEmit": true,
+      "moduleResolution": "node",
```

Comment:
I believe `NodeNext` or `Node16` should be used, since `node` is a pretty old resolution strategy.

See https://www.typescriptlang.org/tsconfig/#moduleResolution